portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,15 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 /**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+Author: Yi Lyu
+Date:   2020.07.08
+Email:  isabella_aus_china@outlook.com
+*/
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// Jump to intro page from start page
+function jumpToIntro() {
+    window.location.href="intro.html";
```

Comment:
Nit: Maybe make it an absolute path? Like /intro.html.

Not sure about this, but it looks to me that this is a relative URL navigation, though you don't have hierarchical structure for now, you may encounter them later. Just to make sure that you know the difference between relative URL navigation and absolute URL navigation.